gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageTestUtils.java

```diff
@@ -28,6 +30,7 @@
 public final class GoogleCloudStorageTestUtils {
 
   public static final HttpTransport HTTP_TRANSPORT = new NetHttpTransport();
+  public static final JsonFactory JSON_FACTORY = GsonFactory.getDefaultInstance();
```

Comment:
Seems like we can delete this code

Reply:
this is needed by the added test, the test is marked ignored as it fails 
https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/820/files#diff-9323db8d8583cefd2a02a3a84dfd7b5139e4ebb5c97a4b9f263dc556dc471f63R676